### ISSUE TITLE
Custom Emojis do not respect selected file system for uploads

### ISSUE THREAD

**Issue by SudeepParajuli** [RocketChat/Rocket.Chat#5884]
Rocket.Chat Version: 48
Node Version: 4.2.6 
MongoDB shell version: 3.2.11
Mongofiles version: r3.2.11
os : ubuntu:16.04

I have been using the GridFS file system to store my custom emoji and avatars files. I want to import and export these custom emojis and avatar files into my local filesystem using `mongofiles` but I am not able to perform the operation. 

Further, I  moved the existing avatars and custom emojis with `mongofiles` option but it is storing in different files than avatars.chunks , avatars.files and custom_emoji.chunks , custom_emoji.files .

Further, if I create a new database and try this operation it performs as expected. 
Why is rocketchat not providing an option for storing custom emoji and custom avatar in the s3 bucket?

**Comment by engelgabriel** [comment]
@RocketChat/core anyone have done this before?

**Comment by SudeepParajuli** [comment]
@engelgabriel please accept my ignorance but one thing I am not getting is why we have not made avatars and emoji to store in s3 bucket ? Is this in the bucket list to release in the future version ?

**Comment by engelgabriel** [comment]
Yes @SudeepParajuli , it was just lack of time to implement the same logic to all packages

**Comment by SudeepParajuli** [comment]
Looking forward to get support from Rocketchat community if there is any workaround for this . By the way upto when could we expect the s3 bucket option ? @engelgabriel

**Comment by graywolf336** [comment]
@Hudell I renamed this pull request to reflect the new issue based upon your comment
